tests/test_add.py

```diff
@@ -415,7 +415,9 @@ def test(self):
             ret = main(["add", self.FOO])
             self.assertEqual(0, ret)
 
-            foo_stage = os.path.abspath(self.FOO + Stage.STAGE_FILE_SUFFIX)
+            foo_stage = os.path.relpath(
+                self.FOO + Stage.STAGE_FILE_SUFFIX, os.getcwd()
```

Comment:
```suggestion
                self.FOO + Stage.STAGE_FILE_SUFFIX
```

relpath doesn't need os.getcwd(), it is what it uses by default